packages/overdrive/lib/components/SelectInput/SelectInput.tsx

```diff
@@ -1,38 +1,56 @@
 import { ChevronDownIcon } from '@autoguru/icons';
 import * as React from 'react';
 import { ReactNode } from 'react';
+import { useStyles } from 'react-treat';
 
-import { Box, useBoxStyles } from '../Box';
+import { Box } from '../Box';
 import { Icon } from '../Icon';
 import { withEnhancedInput } from '../private/InputBase';
+import * as styleRefs from './SelectInput.treat';
 
 export const SelectInput = withEnhancedInput<
 	{
 		children: ReactNode[];
+		c;
 	},
 	HTMLSelectElement
 >(
-	({ field, eventHandlers, suffixed, prefixed, validation, ...rest }) => (
-		<Box
-			display="flex"
-			flexWrap="nowrap"
-			alignItems="center"
-			justifyContent="center">
+	({ field, eventHandlers, suffixed, prefixed, validation, ...rest }) => {
+		const styles = useStyles(styleRefs);
+		return (
 			<Box
-				is="select"
-				flexGrow={1}
-				{...eventHandlers}
-				{...field}
-				{...rest}
-				autoComplete="off"
-			/>
-			<Icon
-				size="medium"
-				icon={ChevronDownIcon}
-				className={useBoxStyles({ marginRight: '4', flexShrink: 0 })}
-			/>
-		</Box>
-	),
+				display="flex"
+				flexWrap="nowrap"
+				alignItems="center"
+				justifyContent="center"
+				position="relative">
+				<Box
+					is="select"
+					flexGrow={1}
+					{...eventHandlers}
+					{...field}
+					{...rest}
+					className={[
+						styles.input,
+						styles.paddedInput,
```

Comment:
Actually, think we can make this simply be a box prop no?

`<Box paddingRight="8">`

Reply:
@maraisr I needed to put a comment in there perhaps but that's the simplest most none-intrusive way I could think of to overpower default padding applied by withEnhancedInput. This is to get two classnames applied to that right padding